src/cms/README.md

```diff
@@ -6,12 +6,20 @@ Strapi allows administration of site data.
 Run a postgres instance
 eg: docker run --name postgres-docker -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres
 
-## Dev steps to run locally
+## Dev steps to initialize locally
 
 1. Install and activate NodeJS 14.
-2. Install and start up postgres locally. You'll need to create a user and database,
-   and set ``DATABASE_HOST``, ``DATABASE_PORT``, ``DATABASE_NAME``, ``DATABASE_USERNAME``,
+2. Install and start up postgres locally. You'll need to create a user and database
```

Comment:
Step 2 isn't necessary if you run the postgres container with `POSTGRES_DB=cms` env

E.g., `docker run --name postgres-docker -e POSTGRES_PASSWORD=postgres -p 5432:5432 -e=POSTGRES_DB=cms -d postgres`

Reply:
[c58f1c3](https://github.com/bcgov/bcparks.ca/pull/258/commits/c58f1c37ae15505a413ecb03b7ab69bcb3e51529)